### PR TITLE
Refactor/spec overhaul

### PR DESCRIPTION
**Pull request by aelydens**

### CODE REVIEW

**Comment by aelydens** [comment]
@westonplatter just doing some user spec cleanup plus dropping a column (is_participating_next_month) that never gets used. give it a once-over when you have a minute!

File: spec/models/mentor_spec.rb [hunk 1]
```diff
@@ -12,9 +12,9 @@
...
+ 			create = User.create!(email: "hellowerqerwred2@gmail.com", password: "Somethingwierd12",password_confirmation: "Somethingwierd12", first_name: "Hello",last_name: "world", mentor: true, primary_industry: "Technology", stage_of_career: 5, mentor_industry: "Technology", peer_industry: ["Business", "Technology", "Startup"].sample, current_goal: ["Rising the ranks / breaking the glass ceiling","Switching industries","Finding work/life balance"].sample,top_3_interests: ["Arts", "Music", "Crafting", "Home improvement / Decorating", "Being a mom", "Dogs", "Cats", "Watching Sports", "Outdoors / Hiking", "Exercise", "Biking", "Yoga", "Running", "Beer","Wine","Traveling"," Local events",    "Reading", "Photography", "Movies","Cooking / Eating / Being a foodie" ,"Social issues / volunteering","Video Games"].sample(3), live_in_detroit: %w(true false).sample, is_participating_this_month: true,
```
Comment (rebeccafae) [inline spec/models/mentor_spec.rb:15]: Could you multi-line these? You could also you Factorygirl to create them

File: spec/models/mentor_spec.rb [hunk 2]
```diff
@@ -27,11 +27,11 @@
...
+ 			user = User.create!(email: "hello324q234q3242@gmail.com", password: "Somethingwierd12",password_confirmation: "Somethingwierd12", first_name: "Hello",last_name: "world", mentor: true, primary_industry: "Technology", stage_of_career: 5, mentor_industry: "Technology", peer_industry: ["Business", "Technology", "Startup"].sample, current_goal: ["Rising the ranks / breaking the glass ceiling","Switching industries","Finding work/life balance"].sample,top_3_interests: ["Arts", "Music", "Crafting", "Home improvement / Decorating", "Being a mom", "Dogs", "Cats", "Watching Sports", "Outdoors / Hiking", "Exercise", "Biking", "Yoga", "Running", "Beer","Wine","Traveling"," Local events",    "Reading", "Photography", "Movies","Cooking / Eating / Being a foodie" ,"Social issues / volunteering","Video Games"].sample(3), live_in_detroit: %w(true false).sample, is_participating_this_month: true,
```
Comment (rebeccafae) [inline spec/models/mentor_spec.rb:30]: This seems to be the same as the user in the test above. Have you thought about moving it to a `let` block?

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -1,92 +1,162 @@
...
-  		user.update(top_3_interests: [])
-  		expect(user.waitlist).to eq(true)
-  	end
+  describe '.update_month' do
+    #update_month 'refreshes' user settings for next month
+
+    context 'if user is participating this month' do
+      let!(:participating_user) do
+        create(:skinny_user,
+                           is_participating_this_month: true,
```
Comment (rebeccafae) [inline spec/models/user_spec.rb]: This is odd [indentation](https://github.com/bbatsov/ruby-style-guide#no-double-indent)

File: spec/models/mentor_spec.rb (hunk 1 above)
Comment (aelydens) [reply spec/models/mentor_spec.rb:15]: This is not a refactor of mentor spec. I merely removed a field on user that was not being used, and adjusted the other specs accordingly.

**Review by rebeccafae — APPROVED** [review]

File: spec/models/user_spec.rb (hunk 3 above)
Comment (aelydens) [reply spec/models/user_spec.rb]: Thanks for catching. I just switched text editors, so I think me getting used to new shortcuts resulted in a few errant spaces. No need to link indentation styleguide, but thanks. :)

**Comment by aelydens** [comment]
Merging! thanks for the review, @rfjackson11

**Pull request closed without merging**